src/types/interfaces.ts

```diff
@@ -34,6 +34,7 @@ export interface ArchipelagoController {
   getPeersData(ids: string[]): Promise<Record<string, PeerData>>
   dispose(): Promise<void>
   flush(): Promise<void>
```

Comment:
Flush returns a promise even though it could return `void`? Was this on purpose? Should we do the same for `modifyOptions`?

Reply:
Hmmm... I think I defined the interface before the implementation, and was thinking that maybe we could return a promise that was resolved when the flush was completed.

But since I didn't do that, we should probably change flush to void to avoid confusion :sweat_smile:

Reply:
Done!